components/edge-functions/spam-classifier.mdx

```diff
@@ -0,0 +1,403 @@
+---
+title: Spam Classifier
+description: High-performance spam detection component using machine learning at the edge for real-time content classification.
+---
+
+import EdgeeSdk from '/snippets/edgee-sdk.mdx';
+
+<EdgeeSdk />
+
+Find it on GitHub: [<Icon icon="github" iconType="solid" /> /edgee-cloud/spam-classifier-component](https://github.com/edgee-cloud/spam-classifier-component)
+
+The Spam Classifier component is a high-performance machine learning edge function that provides real-time spam detection using a Naive Bayesian classifier.
+This WASM-based component runs at the edge, offering fast and accurate content classification without requiring external services or additional backend infrastructure.
```

Comment:
```suggestion
This Wasm-based component runs at the edge, offering fast and accurate content classification without requiring external services or additional backend infrastructure.
```

Reply:
Is "Wasm" the actual casing for shortening WebAssembly?
I would have thought that WASM would be okay too since it expands to WebAssembly as an acronym